hardhat.config.ts

```diff
@@ -0,0 +1,34 @@
+/** @type import('hardhat/config').HardhatUserConfig */
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@openzeppelin/hardhat-upgrades";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const { PRIVATE_KEY, ETHERSCAN_API_KEY } = process.env;
+
+const config: HardhatUserConfig = {
+  defaultNetwork: "hardhat",
+  networks: {
+    hardhat: {},
+    sepolia: {
+      url: "https://ethereum-sepolia.blockpi.network/v1/rpc/public",
+      accounts: PRIVATE_KEY ? [PRIVATE_KEY] : [],
+    },
+  },
+  solidity: {
+    version: "0.8.21",
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 100000,
```

Comment:
Why did you choose to set higher runs instead of the default one?

Reply:
It represents the trade-off between the deploying code size and code execution cost.
If we have higher runs there, it would reduce the code execution gas cost but a higher deploying gas cost.
In our case, it's good to use higher value there at least in my opinion.

Reply:
We have simple logic here, and can go with '200' runs.

Reply:
Okay, we can change the runs if our logic gets complicated :)